tests/test_wave_05.py

```diff
@@ -116,35 +130,30 @@ def test_delete_goal(client, one_goal):
     # Assert
     assert response.status_code == 200
     assert "details" in response_body
-    assert response_body == {
-        "details": 'Goal 1 "Build a habit of going outside daily" successfully deleted'
-    }
+    assert response_body == {"details": 'Goal 1 "Build a habit of going outside daily" successfully deleted'}
 
     # Check that the goal was deleted
     response = client.get("/goals/1")
     assert response.status_code == 404
+    assert Goal.query.get(1) == None
```

Comment:
I get the error `NameError: name 'Goal' is not defined` when I run this test and it gets to this line. We need to import `Goal` at the top of this file here!